Examples/PINRemoteImage/Pods.WORKSPACE

```diff
@@ -12,13 +12,13 @@ new_pod_repository(
 
 new_pod_repository(
   name = "PINOperation",
-  url = "https://github.com/pinterest/PINOperation/archive/1.1.zip",
+  url = "https://github.com/pinterest/PINOperation/archive/1.2.1.zip",
   generate_header_map = True
 )
 
 new_pod_repository(
   name = "PINCache",
-  url = "https://github.com/pinterest/PINCache/archive/d886490de6d297e38f80bb750ff2dec4822fb870.zip",
+  url = "https://github.com/pinterest/PINCache/archive/3.0.3.zip",
```

Comment:
Fixing https://github.com/pinterest/PodToBUILD/issues/173

The older version of PINCache does not compile in Xcode 12.